pkg/virt-config/featuregate/active.go

```diff
@@ -74,3 +74,38 @@ const (
 	// through the kv.spec.configuration.instancetype.referencePolicy configurable.
 	InstancetypeReferencePolicy = "InstancetypeReferencePolicy"
 )
+
+func init() {
+	RegisterFeatureGate(FeatureGate{Name: ExpandDisksGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: CPUManager, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: IgnitionGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: HypervStrictCheckGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: SidecarGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: HostDevicesGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: SnapshotGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VMExportGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: HotplugVolumesGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: HostDiskGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VirtIOFSGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: DownwardMetricsFeatureGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: Root, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: ClusterProfiler, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: WorkloadEncryptionSEV, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VSOCKGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: DisableCustomSELinuxPolicy, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: KubevirtSeccompProfile, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: DisableMediatedDevicesHandling, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: PersistentReservation, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VMPersistentState, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: MultiArchitecture, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VMLiveUpdateFeaturesGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: BochsDisplayForEFIGuests, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: AutoResourceLimitsGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: AlignCPUsGate, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VolumesUpdateStrategy, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: VolumeMigration, State: Alpha})
+	RegisterFeatureGate(FeatureGate{Name: NodeRestrictionGate, State: Alpha})
+
+	RegisterFeatureGate(FeatureGate{Name: NetworkBindingPlugingsGate, State: Beta})
+	RegisterFeatureGate(FeatureGate{Name: DynamicPodInterfaceNamingGate, State: Beta})
```

Comment:
These two were GAed by https://github.com/kubevirt/kubevirt/pull/13314 and https://github.com/kubevirt/kubevirt/pull/13243.

Reply:
Ahh... the rebase messed it up :(